product_category_company/security/ir_rule.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
```

Comment:
Are we sure `noupdate="1"` is correct here?

Reply:
You're right, please remove noupdate, thanks @carmenbianca